Makefile

```diff
@@ -20,6 +20,22 @@ QEMU ?= qemu-system-x86_64 -machine microvm,x-option-roms=on,pit=off,pic=off,rtc
         -enable-kvm -cpu host,invtsc=on,kvmclock=on -no-reboot \
         -m 4G -nographic -monitor none -serial stdio \
         -device virtio-net-device,netdev=net0 -netdev tap,id=net0,ifname=tap0,script=no,downscript=no
+
+# emulate cloud VM (ops onprem)
```

Comment:
Is "cloud VM" a generic established term or does it refer to a specific Cloud architecture?

Reply:
Fixed in 872acfc32c4def8a7e42185cf90dcad79d0cacc8